ts/Extensions/Annotations/Types/Measure.ts

```diff
@@ -275,32 +279,29 @@ function min(
             this.yAxisMax
         );
 
-    let min: (''|number) = Infinity,
+    let min: number = Infinity,
         isCalculated = false; // To avoid Infinity in formatter
 
-    series.forEach((serie): void => {
+    series.forEach((s): void => {
         if (
-            serie.visible &&
-            serie.options.id !== 'highcharts-navigator-series'
+            s.visible &&
+            s.options.id !== 'highcharts-navigator-series'
         ) {
-            serie.points.forEach((point: Point): void => {
+            s.points.forEach((point: Point): void => {
                 if (
-                    !point.isNull &&
-                    (point.y as any) < min &&
-                    (point.x as any) > ext.xAxisMin &&
-                    (point.x as any) <= ext.xAxisMax &&
-                    (point.y as any) > ext.yAxisMin &&
-                    (point.y as any) <= ext.yAxisMax
+                    isNumber(point.y) &&
```

Comment:
already checked in `isPointWithinExtremes`

Reply:
Yeah, I've tried that but then it complains that `point.y > max` could be null or undefined
![image](https://github.com/user-attachments/assets/5de2c039-0e34-42a7-ad81-021e128f9d3e)

Reply:
Oh, I see. Thanks for checking!